docs/tools/testing/icetea.md

```diff
@@ -0,0 +1,188 @@
+## Icetea test framework
+
```

Comment:
What sort of line endings do you have here? They look like they're displaying a bit oddly.